bin/apisix

```diff
@@ -507,6 +507,7 @@ local function init()
         with_module_status = with_module_status,
         node_ssl_listen = 9443,     -- default value
         error_log = {level = "warn"},
+        enable_reuseport = true, -- default true
```

Comment:
the default value should be set in `config.yaml`, not here.

Reply:
done